App/app.rb

```diff
@@ -18,10 +22,13 @@ def select_option(user_choice)
       @people_options.list_all_people
     when '3'
       @people_options.create_person
+      Storage.save_date('Person', @people_options.people_list)
     when '4'
       @book_options.create_book
+      Storage.save_date('books', @book_options.books_list)
     when '5'
       @rentals_list.create_rental
+      Storage.save_date('Rental', @rentals_list.rentals_list)
     when '6'
```

Comment:
[OPTIONAL] Kindly check all spellings to make sure you are not making a typo mistake. For instance, check if you do have a method `save_date`and not `save_data`. :stuck_out_tongue_winking_eye:

Reply:
@jerryowusu if please check the commit its already fixed [af28956](https://github.com/myaserkhan/OOP-school-library/pull/6/commits/af28956b2e48ddb22c9b65ce96bd4603ab1eb213)